### ISSUE TITLE
Recordings labelled as "(Set)"

### ISSUE THREAD

**Issue by T3hlo** [AmsterdamUMC/AmsterdamUMCdb#60]
Some numeric recordings which are not lab results have an addon "(Set)". These look to me as if they are all ventilator settings. Could someone please confirm if this is a correct interpretation?

**Comment by patrickthoral** [comment]
Yes, that's correct. The `(Set)` label is often used to make the distinction between settings and measurements of medical devices. Please also have a look at the [Dictionary](https://github.com/AmsterdamUMC/AmsterdamUMCdb#amsterdamumcdbget_dictionary). It contains descriptions for commonly used ICU parameters. If you don't want to install the python package you can also directly get the csv file [here](https://raw.githubusercontent.com/AmsterdamUMC/AmsterdamUMCdb/master/amsterdamumcdb/dictionary/dictionary.csv).
